evgMrmApp/Db/Makefile

```diff
@@ -31,10 +31,10 @@ DB += cpci-evg-300.db
 DB += vme-evg230.db
 DB += vme-evg230-nsls2.db
 DB += mtca-evm-300.db
-ifeq ($(shell echo "$(BASE) > 3.14" | bc),1)
+ifndef BASE_3_14
```

Comment:
fyi. the intended scheme for using these `BASE_X_Y` macros can be found in [configure/CONFIG_BASE_VERSION](https://github.com/epics-base/epics-base/blob/b41787b6bf8093caaf00e934b6a132963824fbe7/configure/CONFIG_BASE_VERSION#L18-L37).

A condition `> 3.14` should be expressed as

```make
ifdef BASE_3_15
```

Strictly speaking `ifndef BASE_3_14` is expressing `< 3.14`.  However, Base 3.13 used a different set of Makefiles, which mrfioc2 never supported.

Reply:
Ok.